mule-gateway/modules/ROOT/pages/policies-custom-getting-started.adoc

```diff
@@ -141,7 +141,7 @@ is implemented with the following configuration:
 before jumping to the next, and any element placed after that will be executed after returning from the next.
 
 Now that the project is set, you can update the logic as necessary with your requirements. +
-The policy engine is powerful and enables you to do things like modifying HTTP requests and responses, logging, caching, throttling, and almost any other integration that you can do with the Mule runtime engine.
+The policy engine is powerful and enables you to do things like modifying HTTP requests and responses, logging, caching, throttling, and almost any other Mule runtime engine integration.
```

Comment:
```suggestion
The policy engine is powerful and enables you to do things like modifying HTTP requests and responses, logging, caching, throttling, and almost any other Mule runtime engine integration.
```

I don't understand what this bit means in this context: "and almost any other Mule runtime engine integration"

Reply:
I don't understand either, so I just reverted my changes from this topic :)